compiler-plugin/src/main/java/io/ballerina/transformer/plugin/diagnostic/DiagnosticMessage.java

```diff
@@ -58,7 +58,12 @@ public enum DiagnosticMessage {
             DiagnosticSeverity.ERROR),
     ERROR_110("TRANSFORMER_ERROR_110",
             "Unknown Module level entity detected.",
-            DiagnosticSeverity.ERROR);
+            DiagnosticSeverity.ERROR),
+
+    HINT100("TRANSFORMER_ERROR_100",
+            "Since the transformer function {0} is not an 'isolated' function, " +
+                    "no concurrent calls will be made to that function",
```

Comment:
How about `Counterpart resource function of the transformer function {0} will not support concurrent calls. Therefore consider making it 'islolated'.` ?

Reply:
Do we have to mention about the resource function?